modules/bootkube/resources/manifests/01-image-registry-namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
I don't understand the distinction between `modules/bootkube` and `modules/tectonic`, and [we consumer them both together][1], so maybe there isn't a distinction.  @abhinavdahiya, @yifan-gu, do either of you know?

On the other hand, this will all be handled differently soon, so I'm fine with this PR landing without agonizing over the target location ;).

[1]: https://github.com/openshift/installer/blob/master/steps/assets/base/tectonic.tf#L11-L78

Reply:
IIRC the only difference was that anything in bootkube used the binary process for object creation (which you dig into in comment below) - and was vaguely thought of as "anything required to get the cluster running". Anything in tectonic had a bash script that created the objects in a hard-coded order - and were all the "on top of" components (e.g. monitoring). They probably could/should be collapsed at some point - but in near term likely doesn't matter.

Reply:
From in-person discussion with @abhinavdahiya he would prefer that anything not directly required to get a working control-plane ends up under the tectonic module for now - so I believe this should be added to: https://github.com/openshift/installer/blob/741d5cc084e2e22730268c71c6ddfc97fec19c62/modules/tectonic/resources/tectonic.sh#L130-L171